news/rustconf-2024-what-to-expect.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'RustConf 2024: What to Expect'
+byline: Rust Foundation Team
+description: 'RustConf 2024 is less than a month away. Here is what to expect. '
+date: 2024-08-21T12:00:00Z
+tags:
+  - announcement
+  - foundation
+  - rustconf
+index: false
+layout: layouts/news.njk
+---
+This year, the Rust Foundation is excited to host[<u>RustConf</u>](https://rustconf.com/) for the first time in Montreal, Canada, and online from September 10-13, 2024. Less than one month remains!
+
+## What to Expect
+
+RustConf 2024 will feature talks covering various topics within open source and Rust.
+
+Some key themes include:
+
+* **Security:** Rust’s role in enhancing security, covering topics such as making open source secure by design, memory safety across foreign function boundaries, and “X-safety” in unsafe environments.
```

Comment:
Change to: **Security:** Rust’s role in enhancing security, including making open source secure by design, memory-safety across foreign function boundaries, and “X-safety” in unsafe environments.